src/framework/constraints.ml

```diff
@@ -441,7 +458,7 @@ struct
 
   let paths_as_set ctx =
     let liftmap = List.map (fun x -> D.lift x) in
-    lift_fun ctx liftmap S.paths_as_set (Fun.id) []
+    lift_fun ctx liftmap S.paths_as_set (Fun.id) [D.bot ()]
```

Comment:
Why is this change of the default value needed here?

Reply:
Without it many of the longjmp tests fail because `combine_env` of the entire `Spec` is never called if the function's normal return node is dead (and thus the list is empty). But it needs to be delegated because there could be a longjmpy return that's not dead.